### PR TITLE
Don't include download link for replays without/with bad replay files on replay tables

### PR DESCRIPTION
**Pull request by 32th-System**

### CODE REVIEW

File: .github/workflows/deploy_staging.yml [hunk 1]
```diff
@@ -7,7 +7,9 @@ on:
 
 jobs:
   request_deployment:
-
+    
+    if: ${{ secrets.STAGING_DEPLOYER_PASSWORD }}
```
Comment (n-rook) [inline .github/workflows/deploy_staging.yml:11]: This seems unrelated, send out a separate PR

File: project/thscoreboard/replays/templates/replays/replay_table.html [hunk 2]
```diff
@@ -50,8 +50,12 @@
             <td>
                 {{replay.created | date:"Y-m-d"}}
             </td>
-            <td>
+            <td style="text-align:center;">
+            {% if replay.is_good %}
```
Comment (n-rook) [inline project/thscoreboard/replays/templates/replays/replay_table.html:54]: This makes sense to me, good idea!

**Review by n-rook — APPROVED** [review]

File: .github/workflows/deploy_staging.yml (hunk 1 above)
Comment (n-rook) [reply .github/workflows/deploy_staging.yml:11]: Wait you already did lol

Comment (32th-System) [reply .github/workflows/deploy_staging.yml:11]: wait did I forget that I did this PR and the other one on the same main branch on my fork. lol
